kyo-scheduler/jvm/src/main/scala/kyo/scheduler/util/MovingStdDev.scala

```diff
@@ -0,0 +1,39 @@
+package kyo.scheduler.util
+
+final private[kyo] class MovingStdDev(window: Int):
```

Comment:
Since this class isn't used in a very hot path anymore and is called only by regulators that execute periodically, it was updated to favor precision over performance.